pallets/loans/src/entities/pricing/external.rs

```diff
@@ -67,6 +67,11 @@ pub struct ExternalPricing<T: Config> {
 
 	/// Reference price used to calculate the interest
 	pub notional: T::Balance,
+
+	/// Maximum variation between the settlement price chosen for
+	/// borrow/repay and the current oracle price.
+	/// Represented as: Oracle price +/- oracle price * max_variation_price
+	pub max_variation_price: T::PerThing,
```

Comment:
Finally, chosen variation, because *delta* usually means a "variation" but in the same units, i.e: $x + \Delta x$, and what we have is measured in %

Reply:
Nit but then I think `max_price_variation` is more correct still :)

Reply:
BTW, in the chat when you wrote:
*"I think “max delta price” is the more correct English order"*
did you want to say the opposite?

Regarding the "time" meaning for variation, maybe "proportion" could fit better